packages/gluestick-plugin-bunyan/src/server.js

```diff
@@ -19,24 +20,41 @@ export type Logger = {
   error: Function,
 };
 
+type PluginUtilities = {
+  requireModule: Function,
+  defaultLogger: Logger,
+};
+
 const defaultSettings: Options = {
   name: 'default name',
 };
 
-const bunyanPlugin = (): { logger: Logger } => {
-  let options: Options = defaultSettings;
+const bunyanPlugin = (
+  opts: Object,
+  { requireModule, defaultLogger }: PluginUtilities,
+): { logger?: Logger } => {
+  let options: Options = { ...defaultSettings };
```

Comment:
Is there any reason to make a shallow copy here? Nothing in this file is changing it.